sewer/cli.py

```diff
@@ -127,6 +136,10 @@ def main():
     endpoint = args.endpoint
     email = args.email
     loglevel = args.loglevel
+    out_dir = args.out_dir
+
+    if not os.access(out_dir, os.W_OK):
+        raise OSError("The dir '{0}' is not writeable".format(out_dir))
```

Comment:
This is awesome. Thanks

Reply:
Sorry, typo here. writeable should be writable.